REQUIRE

```diff
@@ -4,7 +4,7 @@ Distributions
 Docile
 DualNumbers
 FITSIO
-ForwardDiff 0.0.3 0.1
+ForwardDiff # Currently only required for testing.
```

Comment:
Yes, I'm saying it's not immediately obvious that the ForwardDiff API will suffer the contortions that the input vector takes as it passes through Transform.jl and ElboDeriv.  But I'm newly optimistic.  In any case, we can always just use our own FD implementation using the old DualNumbers package, so I'm sure we can get rid of GSL.jl.  I'll let you know what I find out.